godot-core/src/docs.rs

```diff
@@ -79,20 +97,23 @@ pub fn gather_xml_docs() -> impl Iterator<Item = String> {
 
             let InherentImplDocs {
                 methods,
-                signals,
-                constants,
+                signals_block,
+                constants_block,
             } = pieces.inherent;
 
             let virtual_methods = pieces.virtual_methods;
-            let brief = description.split_once("[br]").map(|(x, _)| x).unwrap_or_default();
+            let methods_block: String = if methods.is_some() || virtual_methods.is_some() { format!("<methods>{m}{vm}</methods>", m=methods.unwrap_or_default(), vm=virtual_methods.unwrap_or_default()) } else { String::new()};
+
+        let brief = description.split_once("[br]").map(|(x, _)| x).unwrap_or_default();
+
```

Comment:
I fixed the formatting of this in my commit, but you reverted it now. Please restore.